### ISSUE TITLE
nodepurejs auth interop_to_prod tests failing on master

### ISSUE THREAD

**Issue by jtattermusch** [grpc/grpc#20425]
Failing 100% runs on master

https://source.cloud.google.com/results/invocations/314440c3-b91d-4529-b663-f0b54a89b064/targets/github%2Fgrpc%2Finterop_test/tests

```
assert.js:85
  throw new AssertionError(obj);
  ^
AssertionError [ERR_ASSERTION]: Input A expected to strictly equal input B:
+ expected - actual
- ''
+ 'interop-to-prod-tests@grpc-testing.iam.gserviceaccount.com'
    at /var/local/git/grpc-node/test/interop/interop_client.js:428:12
    at Http2CallStream.call.on (/var/local/git/grpc-node/packages/grpc-js/build/src/client.js:93:17)
    at Http2CallStream.emit (events.js:194:15)
    at process.nextTick (/var/local/git/grpc-node/packages/grpc-js/build/src/call-stream.js:75:22)
    at process._tickCallback (internal/process/next_tick.js:61:11)
```

**Comment by murgatroid99** [comment]
I fixed this failure with grpc/grpc-node#1055. As of 2019-10-02, this test job was passing.

**Comment by AspirinSJL** [comment]
Looks green today.
